lib/constants.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const { CLUSTER_BASE_URL } = require('./utils/configurations');
+
+const clusters = {
+  new: 'new',
+  top: 'top'
+};
+
+const collectionPaths = {
+  TOP_FREE: { collection: 'topselling_free', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  TOP_PAID: { collection: 'topselling_paid', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  GROSSING: { collection: 'topgrossing', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  TRENDING: { collection: 'movers_shakers', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  TOP_FREE_GAMES: { collection: 'topselling_free_games', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  TOP_PAID_GAMES: { collection: 'topselling_paid_games', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  TOP_GROSSING_GAMES: { collection: 'topselling_grossing_games', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
+  NEW_FREE: { collection: 'topselling_new_free', path: `${CLUSTER_BASE_URL}/${clusters.new}` },
+  NEW_PAID: { collection: 'topselling_new_paid', path: `${CLUSTER_BASE_URL}/${clusters.new}` },
+  NEW_FREE_GAMES: { collection: 'topselling_new_free_games', path: `${CLUSTER_BASE_URL}/${clusters.new}` },
+  NEW_PAID_GAMES: { collection: 'topselling_new_paid_games', path: `${CLUSTER_BASE_URL}/${clusters.new}` }
+};
+
+module.exports.clusters = clusters;
+
+module.exports.paths = Object.keys(collectionPaths)
```

Comment:
These look suspicious. We should aim to have very obvious and declarative constants, and then if necessary helper functions to process them as needed when they need to be used.

Reply:
Definitely agree with you on this.
So I rewrote the constant module to make it cleaner.